css/main.css

```diff
@@ -88,4 +90,16 @@
 .outer-intro-layer {
   padding-top: 44px;
   padding-left: 32px;
-  padding-bottom: 59px; }
\ No newline at end of file
+  padding-bottom: 59px; }
+
+.registration {
+  background-color: #C4C4C4;
+  padding: 2%; }
+
+.hacker, .mentor, .volunteer {
+  background-color: white;
+  text-align: left;
```

Comment:
the wireframe has the text left-aligned, but wouldn't it be more consistent with other sections if centered?

Reply:
yeah i agree, lets make this text centered then!